### PR TITLE
test: disable flaky tests

### PR DESCRIPTION
**Pull request by lukasholzer**
🎉 Thanks for submitting a pull request! 🎉

#### Summary

Disable flaky tests to bring stability in our CI

---

For us to review and ship your PR efficiently, please perform the following steps:

- [x] Open a [bug/issue](https://github.com/netlify/cli/issues/new/choose) before writing your code 🧑‍💻. This ensures we can discuss the changes and get feedback from everyone that should be involved. If you\`re fixing a typo or something that\`s on fire 🔥 (e.g. incident related), you can skip this step.
- [x] Read the [contribution guidelines](../CONTRIBUTING.md) 📖. This ensures your code follows our style guide and
      passes our tests.
- [x] Update or add tests (if any source code was changed or added) 🧪
- [x] Update or add documentation (if features were changed or added) 📝
- [x] Make sure the status checks below are successful ✅

**A picture of a cute animal (not mandatory, but encouraged)**

### CODE REVIEW

File: tests/command.functions.test.js [hunk 1]
```diff
@@ -643,7 +643,8 @@ test('should serve helpful tips and tricks', async (t) => {
   })
 })
 
-test('should detect file changes to scheduled function', async (t) => {
+// flaky test
+test.skip('should detect file changes to scheduled function', async (t) => {
```
Comment (lukasholzer) [inline tests/command.functions.test.js:647]: FYI @netlify-team-account-1 as it failed randomly on all operating systems

**Review by erezrokah — APPROVED** [review]

**Comment by lukasholzer** [comment]
@erezrokah they are even more flaky now does not seem to be the solution tbh

**Comment by erezrokah** [comment]
> @erezrokah they are even more flaky now does not seem to be the solution tbh

I'm testing https://github.com/netlify/cli/pull/4164 to see if it helps with flakiness too (since we use less resources per machine).

**Comment by erezrokah** [comment]
Closing this per https://github.com/netlify/cli/pull/4164

**Pull request closed without merging**